packages/big-design/src/components/MultiSelect/spec.tsx

```diff
@@ -369,6 +373,29 @@ test('clicking on disabled select options should not trigger onItemClick', async
   await waitForElement(() => screen.getByRole('option', { name: /mex/i }));
 });
 
+test('opening the MultiSelect triggers onOpen', async () => {
+  const { getAllByRole, queryByRole } = render(MultiSelectMock);
+  const button = getAllByRole('button')[2];
+
+  fireEvent.click(button);
+
+  expect(onOpen).toHaveBeenCalled();
+
+  await waitForElement(() => queryByRole('listbox'));
+});
+
+test('closing the MultiSelect triggers onClose', async () => {
+  const { getAllByRole, queryByRole } = render(MultiSelectMock);
```

Comment:
It is recommended to use `screen` instead of using the output of render.

```ts
render(MultiSelectMock);

const button = screen.getAllByRole('button')[2];
```